app/visit/visitService.js

```diff
@@ -109,6 +113,22 @@
         });
     }
 
+
+    function getPatientLastBMI(patient) {
+      var getHeightCM = observationsService.getLastPatientObs(patient, {uuid: HEIGHT_CM}, 'custom:(uuid,display,value,obsDatetime)');
+      var getWeightKg = observationsService.getLastPatientObs(patient, {uuid: WEIGHT_KG}, 'custom:(uuid,display,value)');
+      return $q.all([getHeightCM, getWeightKg])
+        .then(function (result) {
+          var heightCMObs = result[0];
+          var weightKgObs = result[1];
+          var heightM = heightCMObs.value / 100;
```

Comment:
No need to divide by 100, height is provided in meters in the POC UI. The only problem is that RDE allows either meters and centimeters in the input. Can this check whether the value is `m` or `cm` and calculate accordingly?

Reply:
Looks like [HEIGHT (CM)](http://dev.esaude.org:8080/openmrs/dictionary/concept.htm?conceptId=5090) expects the values to be in ```cm```, should we really be saving them in ```m```? How to tell which one is being used? There is also this issue https://github.com/esaude/esaude-emr-poc/issues/361.

Reply:
I know but a requirements from one of the previous system reviews said that height should be on `m` in order to align with the paper forms. I don't really know why it's `cm` in RDE! Anyway we have to deal with it :unamused: